proto/planning/generate_id.proto

```diff
@@ -7,12 +7,13 @@ package proto;
  * context; concretely, a geometric configuration, and a set of optional
  * constraints. */
 message RegisterPlanContextRequest {
-  string package_name = 1;
+  string system_name = 1;
```

Comment:
What does system_name mean?

Reply:
intent is the name of the client system, so `dexai-alfred`, `viam-robot`, etc...